lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -670,6 +670,564 @@ class AuthList
       success: (data) -> cb?(data)
       error: std_ajax_err => @$request_response_error.text gettext "Error changing user's permissions."
 
+class EmailSelectors
+  constructor: (@$container, @$section, params={}) ->
```

Comment:
I'm not seeing `$container` referenced outside of this method; why are we assigning it as an instance variable instead of making it local?